helper.js

```diff
@@ -135,14 +135,18 @@ exports.txInToString = (dir, txInList, isCollateral) => {
 
 exports.txOutToString = (txOutList) => {
   let result = "";
+  let assetOutStr;
   txOutList.forEach((txOut) => {
-    result += `--tx-out "${txOut.address}+${txOut.value.lovelace}`;
+    assetOutStr = ""
+    result += `--tx-out ${txOut.address}+${txOut.value.lovelace}`;
     Object.keys(txOut.value).forEach((asset) => {
       if (asset == "lovelace") return;
-      result += `+${txOut.value[asset]} ${asset}`;
+      assetOutStr += `+${txOut.value[asset]} ${asset}`;
     });
-    result += `" `;
-    txOut.datumHash && (result += `--tx-out-datum-hash ${txOut.datumHash}`);
+    if (assetOutStr)
```

Comment:
Is this check actually necessary? If `assetOutStr` is empty, the concat will remain the same right?

Reply:
If `assetOutStr` is empty, `result` will end with `+""`. If it is acceptable, it's okay to remove the check.

Reply:
mmm that's true, I am not sure if that is a valid syntax. so let's leave it for safe